### ISSUE TITLE
[IMRPOVEMENTS]: 

### ISSUE THREAD

**Issue by HooferDevelops** [typical-developers/oaklands-feedback#13]
### Roblox Username

HooferBevelops

### What do you think can have quality-of-life improvements?

Revamp the Charge System

### How do you think things can be improved?

Completely remove the ability to swing before the charge criteria is met. REQUIRE the charge criteria to hit 100% before you can swing. This would be similar to the chisel & saw, that require 100% to commit with the action.

**Comment by LuckFire** [comment]
Honestly, while the charge system is nice and all, it's just a weird gimmick to make things "unique". This change makes a lot more sense and is consistent with other areas of the game so I like this more

**Comment by UltrasBlade** [comment]
requiring a full charge would be a bad change. if i were to have the best axe in the game, some trees wouldnt need a full charge (deadwood, bamboo, etc.). i often use the current charge system while cutting down trees, as you dont always need a full charge. if this change does get implemented, i feel like it would be better to just have no charge at all (like LT2) and instead have a cooldown depending on which tool you use. for example; tools with a quick charge would have a low cooldown, while tools with a longer charge time would have a higher cooldown.

**Comment by HooferDevelops** [comment]
> requiring a full charge would be a bad change. if i were to have the best axe in the game, some trees wouldnt need a full charge (deadwood, bamboo, etc.). i often use the current charge system while cutting down trees, as you dont always need a full charge. if this change does get implemented, i feel like it would be better to just have no charge at all (like LT2) and instead have a cooldown depending on which tool you use. for example; tools with a quick charge would have a low cooldown, while tools with a longer charge time would have a higher cooldown.

Yeah, so if this is implemented we would change all charge rates for tools, giving them a more balanced ratio.
